content/Objects/Objects.q.md

```diff
@@ -0,0 +1,1016 @@
+
+ - No output is displayed.
+
+ - It prints `This is an eat() method.`.
+
+:::
+
+
+:::
+
+/// type=REPL
+
+```php
+<?php
+    class Person 
+    {
+        public $name = "Diana";
+        
+        public function eat()
+        {
+            echo "This is an eat() method.";
+        }
+    }
+
+    $john = new Person();
+    echo $john->name;
+?>
+```
+/// type=SS, answer=[5]
+
+On line 13, replace the statement `echo $john->name;` with `$john->eat();`. Execute the program. What is its output?
+
+ - It prints `name`.
+
+ - It prints `Diana`.
+
+ - It prints `Charles`.
+
+ - No output is displayed.
+
+ - It prints `This is an eat() method.`.
+
+:::
+
++++
+
+
++++
+
+### Part 2: Knowledge Assessment
+
+/// type=SS, answer=[4]
+
+Which statement best describes an object?
+
+ - It is a basic building block of a method.
+
+ - It is an actual representation of a function.
+
+ - It is a function that performs a specific action.
+
+ - It is an actual representation or instance of a class.
+
+ - It is a blueprint that defines the characteristics and behaviors of all objects of a specific kind.
+
+
+/// type=SS, answer=[4]
+
+Which statement is true about the `new` keyword?
+
+ - It sets a property of a class.
+
+ - It defines a method of an object.
+
+ - It represents a function in a class.
+
+ - It creates a new object as an instance of a class.
+
+ - It accesses the methods and properties of an object.
+
+
+/// type=SS, answer=[5]
+
+What does the object operator `->` do?
+
+ - It sets a property of a class.
+
+ - It defines a method of an object.
+
+ - It represents a function in a class.
+
+ - It creates a new object as an instance of a class.
+
+ - It accesses the methods and properties of an object.
+
++++
+
+
++++
+
+### Part 3: Finding and Fixing Errors
+
+:::
+
+/// type=REPL, readonly=true
+
+```php
+<?php
+    class Person 
+    {
+        public $name = "Diana";
+        
+        public function eat()
+        {
+            echo "This is an eat() method.";
+        }
+    }
+
+    $pObject = new Person();
+    echo $pObject->$name;
+?>
+```
+/// type=SS, answer=[3]
+
+Execute the program. What is its output?
+
+ - It prints `name`.
+
+ - It prints `Diana`.
+
+ - It produces an error.
+
+ - No output is displayed.
+
+ - It prints `This is an eat() method.`.
+
+
+/// type=MS, answer=[1,2]
+
+What are the error messages?
+
+ - Undefined variable: `name` on line number 13
+
+ - Uncaught Error: Cannot access empty property
+
+ - syntax error, unexpected `'echo'` (T_ECHO) on line number 13
+
+ - Use of undefined constant name - assumed `'name'` on line number 13
+
+ - Object of class `Person` could not be converted to string on line number 13
+
+
+/// type=MS, answer=[1,3]
+
+Which statements correctly describe the error?
+
+ - There is a dollar sign `$` in the `name` property on line 13.
+
+ - There is no semicolon `;` after the `echo` statement on line 13.
+
+ - On line 13, the statement `echo $pObject->$name;` is invalid.
```

Comment:
This is happened several times already. I already inform you that if you check about the arrangement of the options according to their lengths please do not just rely on the GitHub viewer since it renders differently than the one rendered in our App reader. Hope this will not occur again. Thanks.
![image](https://user-images.githubusercontent.com/25945790/60632320-8e578100-9e37-11e9-89c8-8f861f17feeb.png)